app/import_functions.go

```diff
@@ -763,7 +763,7 @@ func (a *App) ImportReply(data *ReplyImportData, post *model.Post, teamId string
 
 	var reply *model.Post
 	for _, r := range replies {
-		if r.Message == *data.Message {
+		if r.Message == *data.Message && r.RootId == post.Id {
```

Comment:
This cover a corner case where you are trying to duplicate a conversation with the bulk import (copy the conversation, and them modify the root message, but not the rest of the replies).

Reply:
couldn't understand this use case.. I will DM you to understand more regarding this.. 

We don't need a test to cover this ?

Reply:
Good point, would be a nice to have. I going to add it :)

Reply:
Done! :)